modules/administration/pages/content-lifecycle.adoc

```diff
@@ -120,16 +120,14 @@ When you have created your project, defined environments, and attached sources a
 Building applies filters to the attached sources and clones them to the first environment in the project.
 
 
-[NOTE]
-====
-About patch sharing
-
-When you clone a vendor channel multiple times in your organization (e.g. you have 2 content projects with the same source vendor channels), {productname} does not create new patch clones for each cloned channel. Instead of that, a single patch clone is shared between your cloned channels.
-
-One important consequence is, that if the vendor modifies a patch (e.g. the patch gets retracted, or the patch packages change) and you build one content project, all channels sharing the cloned patch be synchronized with the original (even the channels in other environments of your content project, or other content project channels in your organization).
-
-You can turn the automatic patch synchronization feature off in your organization settings. You can synchronize the patch later manually using the Patch Sync function (menu:Software[Manage > Channels], click the channel you want to synchronize, and follow the menu:Repositories[Sync]). Note that even manual patch synchronization affects all channels sharing the patch.
-====
+You can clone vendor channels multiple times within your organization, which results in having two or more content projects with the same source vendor channels.
```

Comment:
```suggestion
You can clone vendor channels multiple times within your organization, which results in two or more content projects having the same vendor channels.
```

Reply:
In the end I've rewritten this sentence. Its current form does not make sense (it says cloning multiple channels results in multiple projects having the same vendor channel, but it's actually the other way around: 2 projects having same vendor channels results in multiple clones). I've tried to reduce this to bare minimum in order to prevent any unclarities.